tests/test_manager_page.py

```diff
@@ -0,0 +1,136 @@
+import allure
+import pytest
+
+from common.utils import convert_table_data_to_list, create_word_from_number
+from pages.manager_page import ManagerPage
+
+
+@allure.title("Создание нового клиента")
+@allure.description(
+    """
+    Цель: Проверить создание нового клиента
+    
+    Предусловие: 
+        - Открыть браузер
+        
+    Шаги:
+        1. Открыть страницу "https://www.globalsqa.com/angularJs-protractor/BankingProject/#/manager"  
+        2. Нажать кнопку Add Customer
+        3. Заполнить поля First Name, Last Name, Post Code валидными данными
+        4. Нажать кнопку Add Customer под формой регистрации
+        5. Проверить появление уведомления об успешном создании клиента
+        6. Нажать кнопку Close в окне уведомления
+        7. Нажать кнопку Customers
+        8. Проверить добавление нового клиента
+        
+    Ожидаемый результат:
+        - В списке клиентов появился новый клиент с заданными именем и фамилией     
+    """
+)
+@pytest.mark.parametrize("surname, post_code",
+                         [
+                             ("rand_str", "rand_num")
+                         ])
+def test_add_customer_successfully(request, browser, surname, post_code):
+    post_code_result = request.getfixturevalue(post_code)
+    surname_result = request.getfixturevalue(surname)
```

Comment:
Достаточно специфичное решение. Мы можем передавать данные из фикстур напрямую в наш тест.
```suggestion
def test_add_customer_successfully(browser, rand_str, rand_num):
    post_code_result = rand_num
    surname_result = rand_str
```